### PR TITLE
Chain spec are built from file

### PR DESCRIPTION
**Pull request by JackPiri**

### CODE REVIEW

**Review by DanieleDiBenedetto — APPROVED** [review]
LGTM ! Please update readme as to clarify the process to build a node to sync to the official testnet, or one for local development purposes

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -183,6 +183,11 @@ pub fn local_config() -> Result<ChainSpec, String> {
 }
 
 pub fn testnet_config() -> Result<ChainSpec, String> {
+    ChainSpec::from_json_bytes(&include_bytes!("../chain-specs/zkverify_testnet.json")[..])
+}
+
+#[allow(dead_code)]
+pub fn testnet_config_build() -> Result<ChainSpec, String> {
```
Comment (la10736) [inline node/src/chain_spec.rs:190]: Why don't use this function for `test-dev` chain name?

**Review by la10736 — CHANGES_REQUESTED** [review]
May is better to introduce a new chain spec `test-dev`

File: node/src/chain_spec.rs (hunk 1 above)
Comment (JackPiri) [reply node/src/chain_spec.rs:190]: Addressed as per your suggestion (just opted for `test_build` naming).

**Review by drgora — APPROVED** [review]
Looks good to me, thanks!

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -186,7 +186,7 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
     ChainSpec::from_json_bytes(&include_bytes!("../chain-specs/zkverify_testnet.json")[..])
 }
 
-#[allow(dead_code)]
+// To be used when building new testnet chain-spec
```
Comment (la10736) [inline node/src/chain_spec.rs:189]: Please use `///` to make this comment also a function documentation

**Review by la10736 — CHANGES_REQUESTED** [review]
Please use `///` to generate documentation too

File: node/src/chain_spec.rs (hunk 2 above)
Comment (JackPiri) [reply node/src/chain_spec.rs]: Modified accordingly.

**Review by ptagl — APPROVED** [review]

**Review by la10736 — APPROVED** [review]
LGTM GREAT!

**Pull request merged**